### PR TITLE
Rdm 7284: Release FW to Artefact Repository

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-7284

### Change description ###

Use Github actions to publish artefacts to hmcts maven repository

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,6 +48,61 @@ jar {
...
+}
+
+publishing {
+    publications {
+        Main(MavenPublication) {
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+            groupId project.group
+            artifactId 'idam-client'
```
Comment (MSancaktutar) [inline build.gradle]: idam-client looks like a copy-paste error. :)

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+version=1.0.10-SNAPSHOT
```
Comment (MSancaktutar) [inline gradle.properties]: We already have a buildNumber in build.gradle. Eventual version released will contain that build number. How would the version as specified here work with the build number there? Asking just to make sure we're good in this aspect. Let's confirm end-to-end the process of a new release once the bintray key is defined and merge onto master starts releasing releases.

Comment (kiran-yenigala-hmcts) [reply gradle.properties]: We don't need this now. when I was working on releasing from local yesterday, I have to add this one. I will delete this file now...

File: build.gradle (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply build.gradle]: Changed this...

**Comment by MSancaktutar** [comment]
Merged it straight after review, as we need to merge to be able to test this one.

**Pull request merged**